components/text Utilities/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Surbhi Kukreti" />
+    <meta name="description" content="text utilities component" />
+
+    <link rel="stylesheet" href="/styleGuide/util.css" />
+    <link rel="stylesheet" href="/styleGuide/main.css" />
+
+    <link rel="stylesheet" href="./textUtil.css" />
+
+    <link rel="stylesheet" href="/prism/prism.css" />
+    <script src="/prism/prism.js" defer></script>
+    <title>Text Utilities</title>
+  </head>
+  <body>
+    <div class="text-show">
+      <div class="pad-left">
+        <h3>Text Sizes</h3>
+        <div class="text-container">
+          <p class="vbig-text">Very Big Text</p>
+          <p class="big-text">Big Text</p>
+          <p class="med-text">Medium Text</p>
+          <p class="small-text">small Text</p>
```

Comment:
may be a text-sm, text-md would be make more sense as sm,md etc are as per the standard